### DISCUSSION TITLE
test files in testsuite/spawn_is_dangerous/

### DISCUSSION THREAD

**Discussion by alaingdl** [gnudatalanguage/gdl#1511] [General]
1/ I would like to bring back to `testsuite/` some files  now in `testsuite/spawn_is_dangerous/`
At least two tests in `spawn_is_dangerous/` which were working before are now broken (regressions have not been detected in real time because working tests have been removed)

Maybe SPAWN is __dangerous__  but SPAWN is in the internal commands of GDL & it is used here in the context of automatic tests, not to write somethings designed to be dangerous.

2/ do we have a simple way to know the path to the running GDL ? It would help to fix some tests now using `../src/gdl` which is no more available.

thanks

**Comment by GillesDuvert** [comment]
just insure that the spawned command is the one corresponding to the target machine, windows or [unix/macOs] . even on OSX some commads may not be present.
